test/test.js

```diff
@@ -117,4 +117,17 @@ describe('vue-loader', function () {
     })
   })
 
+  it('local-scope', function(done) {
+    test({
+      entry: './test/fixtures/localcss.js',
+      vue: {
+        injectStyle: true
+      }
+    }, function (window) {
+      var module = window.testModule
+      expect(module.data().styles.red).to.equal('_3iZItEFWW8HWfKNmM95gGD')
```

Comment:
Can we add 2 assertions here:

1. the data should also contain the original `msg` field;
2. the style has been inserted with the same class name. (possibly by selecting the first <style> node and do a regex match on its textContent)